conditions.py

```diff
@@ -0,0 +1,13 @@
+### practicing the conditional logic in python with if & elif
```

Comment:
GPT summary of [None](https://github.com/SampatSuthar/ml-demo/blob/cccc5e0aa50c29cda8434cc1a92463d295c7584b/conditions.py#None) - [bf2262](https://github.com/SampatSuthar/ml-demo/blob/0c97036f0373bc6c942bcdc96667a24a6851e90d/conditions.py#bf226209ab5d6be0a7f03f737e000c30f8ae5cd7):
* Added code to practice conditional logic with if and elif statements
* Set country and state variables
* Added an if statement to check if country is India, and if state is in a list of states
* Added an elif statement to check if state is Maharashtra
* Added an else statement to set a default petrol price if country is not India